### DISCUSSION TITLE
DataType: Time

### DISCUSSION THREAD

**Discussion by Bernie1502** [Budibase/budibase#3752] [Help]
Hi all,
in my MySQL-DB is a Table with a Column `start_at` with datatype `time`.
In BudiBase is the datatype in the Data-Section as `Date/Time`. Also in new generated Tables in BudiBase there is only `Date/Time` as datatype possible.

In the Frontend i can set the DatePicker element to store Time and it does it :) But it doesn't show the Time anymore. In the DB it is stored as Time-Only. Error in the FrontEnd 

> Invalid date provided: 07:00:00

Is it possible to Format the Date/Time or to implement a datatype `Time` or any other suggestions?

Thanks!

**Comment by mjashanks** [comment]
Hi @Bernie1502 - unfortunately this looks like a bug. I have logged it here: https://github.com/Budibase/budibase/issues/3758

I can't see a good workaround for this at the moment. The time will display correctly inside a `Paragraph` element - not sure if that's any use to you.

**Reply by Bernie1502** [reply]
Thanks @mjashanks for the reply. I'll try it with the `Paragraph` element and wait about the fix ;)
